openfisca_france/parameters/prestations/prestations_familiales/paje/base/apres_2018/majoration_biact_parent_isoles.yaml

```diff
@@ -0,0 +1,6 @@
+description: Majoration pour biactifs et isolés
+reference: ipp
```

Comment:
La référence est noté ici comme venant de l'IPP. Or ce tag est lié au parsing des barèmes IPP donc je pense, sauf si la valeur a été extraite des barèmes IPP, qu'il faut le changer en openfisca.